contracts/eosio.system/include/eosio.system/peer_keys.hpp

```diff
@@ -73,7 +73,7 @@ struct [[eosio::contract("eosio.system")]] peer_keys : public eosio::contract {
 
    /**
     * Returns a list of top-50 producers (in rank order) along with their peer public key if it was
```

Comment:
Comment should say `list of up to top-50 producer`. You may want to also clarify the condition on their votes for why it would cut it off earlier than the 50th producer.